site_config/site.js

```diff
@@ -51,19 +51,14 @@ export default {
               key: 'docs7',
               text: '1.1.0(Not Apache Release)',
               link: 'https://analysys.github.io/easyscheduler_docs_cn/',
-          }
-        ]
+          },
+        ],
       },
       {
         key: 'download',
         text: 'DOWNLOAD',
         link: '/en-us/docs/release/download.html',
       },
-      {
```

Comment:
One question is whether it needs to be removed? (I don't know the discussion context, so this is just a question)

Reply:
> One question is whether it needs to be removed? (I don't know the discussion context, so this is just a question)

I also confused about moving developer module， is it needed？

Reply:
Moving how to contribute from the devleopment module to the community module has discussed in the dev email list.

Reply:
The blog module also has removed. What about this?

Reply:
> The blog module also has removed. What about this?
The blog module is deleted because the articles in the blog are quoted in the community dynamics. The developer module has not been deleted at present, the content only retains the developer guide, and the other content has been migrated to the community. In addition, I personally think that the content of our official website is too repetitive. It is necessary to have a discussion together to make the official website refresher. What do you think?
![image](https://user-images.githubusercontent.com/28628088/104810382-15f4bd00-582f-11eb-98a8-8562c4ded07f.png)